src/components/calcite-alert/calcite-alert.e2e.ts

```diff
@@ -216,4 +216,33 @@ describe("calcite-alert", () => {
       expect(await progressBarStyles.getPropertyValue("background-color")).toEqual(overrideStyle);
     });
   });
+
+  describe("when multiple alerts are queued", () => {
+    it("should display number of queued alerts with a calcite-chip", async () => {
+      const page = await newE2EPage({
+        html: `
+        <script>
+          setTimeout(() => {
+            document.querySelector("#alert-to-be-queued").setAttribute("active", "");
+          }, 300);
```

Comment:
The timeout here seems like it has the potential to break easily. Should it use the animationDurationInMs?

Reply:
Should we break this out into an issue? This is something I'm not able to update/fix.

Reply:
@julio8a I can update the test as soon as I'm done helping Manahari debug some git issues

Reply:
updating the test!